erpnext/crm/doctype/lead/lead.py

```diff
@@ -370,3 +374,10 @@ def get_lead_with_phone_number(number):
 	lead = leads[0].name if leads else None
 
 	return lead
+
+def get_mandatory_fields(doc):
```

Comment:
I think that this method can be useful for other purposes. Maybe it should be in a more common file, like utils or something like that.